docs/make.jl

```diff
@@ -33,6 +35,11 @@ const PAGES = [
         "Tensor Products and Partial Traces" => "users_guide/tensor.md",
         "Time Evolution and Dynamics" => [
             "Introduction" => "users_guide/time_evolution/intro.md",
+            "Time Evolution Solutions" => "users_guide/time_evolution/solution.md",
+            "Lindblad Master Equation Solver" => "users_guide/time_evolution/sesolve_and_mesolve.md",
```

Comment:
I would divide the Schrodinger and the Mesolve cases.

Reply:
okay, done ~